datafusion/tests/sql.rs

```diff
@@ -6615,3 +6622,357 @@ async fn csv_query_with_decimal_by_sql() -> Result<()> {
     assert_batches_eq!(expected, &actual);
     Ok(())
 }
+
+#[tokio::test]
```

Comment:
how about add some cases with different TIME_ZONE, for example `UTC-7` or `UTC+8`

Reply:
This change doesn't do any computation with the timezones, it just passes them through from the underlying data.

Reply:
Yeah, among other things I also don't think arrow-rs respects timezones fully (and it doesn't have a full set of arithmetic on time/date/timestamp types either)